### PR TITLE
Update metatags & jobs config

### PR DESCRIPTION
**Pull request by froboy**
**Description**

- Updates metatag config to: 
   - add new share image
   - match `og_title` to `title` on internal pages
   - set `og_title` on homepage 
- update sponsorship links on job board
- update default event

### CODE REVIEW

File: conf/drupal/config/metatag.metatag_defaults.global.yml [hunk 1]
```diff
@@ -8,11 +8,11 @@ id: global
 label: Global
 tags:
   canonical_url: '[current-page:url]'
-  image_src: 'https://www.midcamp.org/themes/custom/hatter/o-midcamp.png'
+  image_src: 'https://www.midcamp.org/sites/default/files/2019-10/MidCamp2020Social.png'
   title: '[current-page:title] | [site:name]'
-  og_image: 'https://www.midcamp.org/themes/custom/hatter/o-midcamp.png'
+  og_image: 'https://www.midcamp.org/sites/default/files/2019-10/MidCamp2020Social.png'
```
Comment (Brian-Clement) [inline conf/drupal/config/metatag.metatag_defaults.global.yml]: Is this PR just waiting on an image to replace here @froboy ?

**Comment by froboy** [comment]
Yup that should be it. I haven’t had time to do the media module work yet.

File: conf/drupal/config/metatag.metatag_defaults.global.yml (hunk 1 above)
Comment (froboy) [reply conf/drupal/config/metatag.metatag_defaults.global.yml]: Yup. Will update once #294 is deployed.

Comment (froboy) [reply conf/drupal/config/metatag.metatag_defaults.global.yml]: @Brian-Clement this should be good to go how. I've uploaded the image using the new media and it just so happened to get the same url. :shipit:

**Pull request merged**